product_docs/docs/pgd/5/tpa/quick_start.mdx

```diff
@@ -19,7 +19,7 @@ architecture using Amazon EC2.
    tpaexec configure myedbdpcluster --architecture PGD-Always-ON --platform aws --location-names eu-west-1 --data-nodes-per-location 3 
    ```
 
-   This creates a subdirectory directory in current working directory called `myedbdpcluster` containing the `config.yml` configuration file TPA uses to create the cluster. Edit the `config.yml` as needed, for example to change the IP address range used for servers or adjust locations of nodes.
+   This creates a subdirectory in current the working directory called `myedbdpcluster` containing the `config.yml` configuration file TPA uses to create the cluster. Edit the `config.yml` as needed, for example to change the IP address range used for servers or adjust locations of nodes.
```

Comment:
Should it be `This creates a subdirectory in **the** current working directory`?

Reply:
yep, thanks for catching that